doc/classes/MeshDataTool.xml

```diff
@@ -7,16 +7,43 @@
 		MeshDataTool provides access to individual vertices in a [Mesh]. It allows users to read and edit vertex data of meshes. It also creates an array of faces and edges.
 		To use MeshDataTool, load a mesh with [method create_from_surface]. When you are finished editing the data commit the data to a mesh with [method commit_to_surface].
 		Below is an example of how MeshDataTool may be used.
-		[codeblock]
+		[codeblocks]
+		[gdscript]
+		var mesh = ArrayMesh.new()
+		mesh.add_surface_from_arrays(Mesh.PRIMITIVE_TRIANGLES, CubeMesh.new().get_mesh_arrays())
 		var mdt = MeshDataTool.new()
 		mdt.create_from_surface(mesh, 0)
 		for i in range(mdt.get_vertex_count()):
 		    var vertex = mdt.get_vertex(i)
-		    ...
+		    # Your edits are going below. In this example we extend the mesh by one unit, which results in seperated faces as it is flat shaded.
```

Comment:
```suggestion
		    # In this example we extend the mesh by one unit, which results in seperated faces as it is flat shaded.
```

This makes the sentence shorter to remove some horizontal scrolling, and I'd say it's understandable from the context that if you want to affect vertices, that's where you should do it.